ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2196,3 +2196,12 @@ def wait_for_sssd_domain_status_online(host, timeout=120):
         time.sleep(5)
     else:
         raise RuntimeError("SSSD still offline")
+
+
+def check_traceback(host):
+    """Checking Traceback  on a remote host.
+    :param host: the host where the check takes place
+    """
+    http_logs = host.get_file_contents(paths.VAR_LOG_HTTPD_ERROR,
+                                       encoding='utf-8')
+    assert 'Traceback' not in http_logs
```

Comment:
This method will not be used anywhere else than `test_idviews.py` testsuite. IMO it should be moved to `test_idviews.py` itself. 
OR 
Make it more generalize here in `tasks.py`. May be  something like:
```
def assert_string(host, string_to_check, file_name):
    """ Check the given string in specified file
    : param host: host where the check takes place
    : param string_to_check: string to be check
    : param file_name: file on which check is to be performed.
    """
    file_contents = host.get_file_contents(file_name)
    assert string_to_check in file_contents
```

Reply:
@mrizwan93 I think we can create separate PR for refactoring work.